client/src/apiClient.ts

```diff
@@ -0,0 +1,13 @@
+import createClient from "openapi-fetch";
+import type { paths, components } from "./apischema"; // generated by openapi-typescript
+export type SimplePollingEvent = components["schemas"]["EventItem"];
+
+const client = createClient<paths>({ baseUrl: "http://localhost:4000" });
+
+export const loadEventsGenerated = async () => {
+  const { data, error } = await client.GET("/api/events", { params: { query: { limit: 10 } } });
```

Comment:
podle tve YAML specifikace by jsi mela mit cestu ne `/api/events` ale jen `/events`, ale podle kodu v serveru naopak opravdu `/api/events`, takze v te specce to mas spatne :)